frmts/stacit/stacitdataset.cpp

```diff
@@ -139,8 +139,7 @@ int STACITDataset::Identify(GDALOpenInfo *poOpenInfo)
             return pszHeader[0] == '{';
         }
 
-        if (strstr(pszHeader, "\"stac_version\"") != nullptr &&
-            strstr(pszHeader, "\"proj:transform\"") != nullptr)
```

Comment:
Why is proj:transform checked for? It could appear both in properties or assets so it seems like this check may lead to false results.

Reply:
The driver requires the projection extension to be there, hence checking for it. And as Identify() needs to be quick (it only has a subset of the JSON and can't use a JSON parser), it doesn't get into the details to figure out where the element is present

Reply:
Oh, now I remember what strstr does. I'll revert, thanks.

Reply:
It needs two out of the three proj: properties and transform is one them. So just checking proj:transform seems to exclude cases where proj:bbox and proj:shape are set. Should we update this to check whether any of them is present?

Reply:
| fields | currently | new | 
| transform + shape | accepted | accepted | 
| transform + bbox | accepted | accepted |
| bbox + shape | rejected | accepted |

Reply:
> Maybe worth a separate small PR though, it's unrelated to the STAC 1.1 updates.

yes